ftw/solr/browser/livesearch.py

```diff
@@ -200,8 +205,21 @@ def get_show_more_item(self):
     def get_nothing_found_item(self):
         label_nothing_found = _('label_nothing_found',
                                 default='No items found')
+        label_suggestion = ''
         params = self.facet_params
-        params += '&SearchableText=' + self.searchterms
+
+        # Use first suggestion in not found label.
+        suggestions = self.suggestions()
+        if suggestions:
+            word, suggestions_params = suggestions[0]
+            label_suggestion = _(
+                'label_nothing_found_suggestions',
+                default='Did you mean..."${word}"',
```

Comment:
I'd also change the english default `'Did you mean..."${word}"'` => `'Did you mean: ${word}'` so that the format is the same as used in other languages.